pkg/daemon/ceph/osd/volume_test.go

```diff
@@ -862,26 +868,244 @@ func TestInitializeBlock(t *testing.T) {
 			}
 
 			// First command
-			if args[9] == "--osds-per-device" && args[10] == "1" && args[11] == "/dev/sda" && args[12] == "--db-devices" && args[13] == "/dev/disk/by-id/nvme-Samsung_SSD_970_EVO_Plus_1TB_XXX" {
-				return `{"vg": {"devices": "/dev/disk/by-id/nvme-Samsung_SSD_970_EVO_Plus_1TB_XXX"}}`, nil
+			if args[9] == "--osds-per-device" && args[10] == "1" && args[11] == "/dev/sda" && args[12] == "--db-devices" && args[13] == "/dev/nvme0n1" {
+				return `{"vg": {"devices": "/dev/nvme0n1"}}`, nil
```

Comment:
When modifying unit tests, it would be best to make a new test. We shouldn't be removing an existing test unless the behavior is changing, and the existing behavior shouldn't be changing, because we need to maintain backwards compatibility. I might suggest copying this test and making these changes in the new test.

Reply:
The reason for the modification here is, the parameter `--db-devices` in  `ceph-volume` will use the device path name, but it will not affect the usage of the upper layer.